recipes/util-linux/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+./autogen.sh
+
+./configure
+
+make
```

Comment:
Please add a terminal newline. Also is there a `make check`? What about `make install`?

Reply:
make check and make install are not in the instructions in the source code repo..